### PR TITLE
Add gitignore to exclude jekyll specific stuff

### PR DESCRIPTION
**Pull request by axilleas**

### CODE REVIEW

**Comment by axilleas** [comment]
Nice change :beers:

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,5 @@
+.bundle
+.sass-cache
+.vendor
+_site
+.dummy
```
Comment (axilleas) [inline .gitignore]: What is this file for?

Comment (axil42) [inline .gitignore]: What's this file for?

Comment (axilleas) [inline .gitignore]: You're right it should not be there.

**Pull request merged**